src/components/dialogs/PublishPollDialog/lang.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright 2017–2018, LaborX PTY
+ * Licensed under the AGPL Version 3 license.
+ */
+
+export const prefix = 'PublishPollDialog'
+
+export default {
+  en: {
+    title: 'Publish this poll?',
+    text: 'You\'re about to publish "%{name}" poll and make it visible to a public. Please confirm blow.',
```

Comment:
'You\'re about to publish "%{name}" poll and make it visible to a public. Please confirm blow.'

Hmmm.. Maybe 'confirm bElow'?